.vscode/extensions.json

```diff
@@ -0,0 +1,8 @@
+{
+  "recommendations": [
+    "dbaeumer.vscode-eslint",
+    "samverschueren.final-newline",
+    "DmitryDorofeev.empty-indent",
+    "esbenp.prettier-vscode"
+  ]
+}
```

Comment:
yeah it's intentional. The editor config is not super useful since we run `prettier` as a pre-commit hook.

Here it's just to recommend some extensions to make working on the codebase easier. I really liked how desktop/desktop does it: https://github.com/desktop/desktop/blob/master/docs/contributing/tooling.md